cdap-ui/app/features/tracker/tracker.less

```diff
@@ -545,6 +544,197 @@ body.state-tracker-help {
   }
 }
 
+body.state-tracker-integrations {
+  background-color: white;
+
+  .integration-item {
+    margin: 0;
+    padding: 10px 25px;
+    border-bottom: 1px solid @border-color;
+    height: 80px;
+    width: 100%;
+    display: table;
+
+    .integration-information {
+      width: 50%;
```

Comment:
the table-cell display is to vertically center elements. The pull-right and pull-left doesn't center the setup buton